app/assets/stylesheets/static_page.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPage controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```

Comment:
If you don't use this file just delete it

Reply:
Sorry,I am facing some issue when reading and looking for suggestion for next force push.kindly read my comment below. Thanks

Reply:
deleted